app/ts-cli/main.go

```diff
@@ -18,14 +18,33 @@ package main
 
 import (
 	"fmt"
+	parse "github.com/influxdata/influxdb/cmd"
+	"github.com/openGemini/openGemini/app"
+	"github.com/openGemini/openGemini/app/ts-cli/cmd"
 	"os"
+	"runtime"
```

Comment:
Packages import order should be like this:

```
Official Libraries/Built-in Packages


Three-party library
```

See [goimports-reviser](https://github.com/incu6us/goimports-reviser)